order.js

```diff
@@ -242,22 +304,59 @@ module.exports = {
       'background-origin',
       'background-clip',
       'background-color',
+      'background-blend-mode',
       'stroke',
       'fill',
+      'paint-order',
```

Comment:
I would probably expect to first set the order and then set the values. This way I would have to do it the other way around. What do you think?